### ISSUE TITLE
password

### ISSUE THREAD

**Issue by geoHeil** [geodocker/geodocker-geomesa#1]
Hi, what is the password for the geodocker accumulo container?
I created a fork to upgrade to 1.8 and play with geomesa https://github.com/geoHeil/geodocker-geomesa but have problems logging in via user root and password GisPwd

What are the correct credentials?

**Comment by pomadchin** [comment]
@geoHeil you set it up during containers launch https://github.com/geodocker/geodocker-geomesa/blob/master/geodocker-accumulo-geomesa/docker-compose.yml#L31

**Comment by geoHeil** [comment]
Indeed, I know that 
        ACCUMULO_PASSWORD: GisPwd
is set to that value. That's why I am using it here https://github.com/geoHeil/geodocker-geomesa

to launch it like

```
java -cp geomesa-quickstart-accumulo/target/geomesa-quickstart-accumulo-1.3.0.jar \
  com.example.geomesa.accumulo.AccumuloQuickStart \
  -instanceId accumulo \
  -zookeepers zookeeper \
  -user root \
  -password GisPwd \
  -tableName quickstart
```

which still fails (after adding the jars for geomesa)
`ERROR: for accumulo-master Cannot start service accumulo-master: oci runtime error: container_linux.go:247: starting container process caused "exec: \"/sbin/geomesa-entrypoint.sh\": permission denied"`

Additionally, I could ovserve that accumulo monitor was not able to obtain the traces due to the same issue.

**Comment by pomadchin** [comment]
@geoHeil i think you have not valid permission for the `entrypoint.sh`, double check that it's executable.

**Comment by jnh5y** [comment]
@geoHeil, @anthonyccri might be able to help.

Which docker images are you running?  @anthonyccri published the GeoMesa GeoDocker and has used them successfully from AWS's EMR.  I haven't had a chance to try them out locally...

**Comment by pomadchin** [comment]
I feel like it can be a problem of building image locally (wrong permissions).
Another problem can be that image is built on mac and mac docker can be buggy sometimes. 

Figured out: potentially it's a typo in a docker file.

**Comment by geoHeil** [comment]
Indeed, I can confirm it is a typo in the entrypoint of the docker file.
